File: First Letter/pig.js
function pig_it(str){
    let value = str.split("")
    console.log(value)

    let newVal = []
    newStr = " ";
    for (let i = 0; i < value.length; i++){
        newStr = value[i].slice(1) + value[i].charAt(0) + "ay";
        newVal.push(newStr); 
    }return newVal.join(" ")
}

console.log(pig_it('Pig latin is cool'))
console.log(pig_it('Hello world!'))

File: First Letter/pig.py
def pig_it(text):
    phrases = text.split(" ")
    new_phrases = []
    for phrase in phrases:
        if phrase.isalpha():
            new_phrase = phrase[1:] + phrase[0] + "ay"
            new_phrases.append(new_phrase)
        else:
            new_phrases.append(phrase)
    return " ".join(new_phrases)

print (pig_it('Pig latin is cool'))
print (pig_it('Hello world !'))